src/main/resources/avro/metadata.avdl

```diff
@@ -9,6 +9,32 @@ protocol Metadata {
 import idl "common.avdl";
 
 /**
+  An ontology term describing an attribute. (e.g. the phenotype attribute
+  'polydactyly' from HPO)
+  */
+  record OntologyTerm {
```

Comment:
There's both a `metadata` and a `metadata-working` branch, each of which has its own version of `OntologyTerm`. Which one should I settle on?